torch/nn/modules/pooling.py

```diff
@@ -1094,6 +1094,20 @@ class AdaptiveMaxPool3d(_AdaptiveMaxPoolNd):
         - Output: :math:`(N, C, D_{out}, H_{out}, W_{out})` or :math:`(C, D_{out}, H_{out}, W_{out})`,
           where :math:`(D_{out}, H_{out}, W_{out})=\text{output\_size}`.
 
+    Behavior:
+        - When the input shape is divisible by the output shape
+          (i.e., `H_in % H_out == 0` and `W_in % W_out == 0`),
```

Comment:
Use double backticks or `:math:` 
```suggestion
          (i.e., :math:`H_in % H_out == 0` and :math:`W_in % W_out == 0`),
```